### PR TITLE
feat(movies): add movies home page UI

### PR DESCRIPTION
**Pull request by mishalisovyi**

### CODE REVIEW

File: src/common/ui/TabBar/TabBar.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    setActiveTabIndex(tabIndex);
+  };
+
+  return (
+    <div className={styles['TabBar']}>
+      {tabsLabels.map((label, index) => {
+        return (
+          <Tab
+            label={label}
+            key={`${label}_${index}`}
```
Comment (danyloburavlov) [inline src/common/ui/TabBar/TabBar.tsx]: `index` should not be used at all as a key.

**Review by danyloburavlov — CHANGES_REQUESTED** [review]

File: src/common/ui/TabBar/TabBar.tsx (hunk 1 above)
Comment (mishalisovyi) [reply src/common/ui/TabBar/TabBar.tsx]: Refactored

File: src/common/ui/TabBar/TabBar.tsx [hunk 2]
```diff
@@ -13,15 +13,19 @@ export function TabBar({ tabsLabels }: TabBarProps) {
     setActiveTabIndex(tabIndex);
   };
 
+  const memoizedHandleTabClick = useCallback((tabIndex: number) => {
```
Comment (danyloburavlov) [inline src/common/ui/TabBar/TabBar.tsx:16]: `const handleTabClick = useCallback((tabIndex: number) => {
  setActiveTabIndex(tabIndex);
}, []);`

**Review by danyloburavlov — CHANGES_REQUESTED** [review]

File: src/common/ui/TabBar/TabBar.tsx (hunk 2 above)
Comment (mishalisovyi) [reply src/common/ui/TabBar/TabBar.tsx:16]: Considered to remain as is

**Review by danyloburavlov — APPROVED** [review]

**Pull request merged**